package.json

```diff
@@ -2,7 +2,7 @@
   "name": "io-ts-reporters",
   "main": "./target/src/index.js",
   "typings": "./target/src/index.d.ts",
-  "version": "0.0.18",
+  "version": "0.0.19",
```

Comment:
Let's remove this change and I'll run a `npm version patch`, which will create this change in a separate commit along with a git tag.